docs/docs/event-brokers.mdx

```diff
@@ -190,6 +195,14 @@ If using the `SASL_SSL` protocol, the endpoints file should look like:
 ```yaml-rasa (docs/sources/data/test_endpoints/event_brokers/kafka_sasl_ssl_endpoint.yml)
 ```
 
+### Sending Events to Multiple Queues
+
+Kafka does not allow you to configure multiple topics. 
+
+However, multiple consumers can read from the same queue as long as they are in different consumer groups.
+Each consumer group will process all events independent of each other 
+(in a sense, each group has their own reference to the last event they have processed).
```

Comment:
I think we should also link to a reference, maybe [this](https://www.oreilly.com/library/view/kafka-the-definitive/9781491936153/ch04.html)?